solidity/dashboard/src/hooks/useSetMaxAmountToken.js

```diff
@@ -0,0 +1,18 @@
+import BigNumber from "bignumber.js"
+import { useFormikContext } from "formik"
+import { toTokenUnit } from "../utils/token.utils"
+
+const useSetMaxAmountToken = (filedName, availableAmount) => {
+  const { setFieldValue } = useFormikContext()
```

Comment:
I totally agree. But can we address it in a cleanup PR? Currently, The PR has 29 changed files. I would like to avoid extra work for reviewers and make a mistake in moving files. 💣 😆